code/modules/modular_computers/computers/subtypes/dev_handheld.dm

```diff
@@ -18,6 +18,6 @@
 	set_icon()
 
 /obj/item/modular_computer/handheld/proc/set_icon()
-	icon_state += pick("", "-blue", "-green", "-red", "-brown")
+	icon_state += ""// pick("", "-blue", "-green", "-red", "-brown")
```

Comment:
just delete this part tbh

Reply:
oh, I forgot that after I finally got it working. My bad.